base_geoengine/geo_model.py

```diff
@@ -18,35 +18,35 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from __future__ import absolute_import
-import json
 
-from osv import fields, osv, orm
-from tools.translate import _
-from . import geo_field
-from . import  geo_db
+from openerp import models
+from openerp.osv import fields, osv
+from openerp.tools.translate import _
 from . import geo_operators
+from . import fields as geo_fields
 
 
-DEFAULT_EXTENT = '-123164.85222423, 5574694.9538936, 1578017.6490538, 6186191.1800898'
+DEFAULT_EXTENT = '-123164.85222423, 5574694.9538936, ' + \
+    '1578017.6490538, 6186191.1800898'
```

Comment:
What do these values represent? It would be nice to have that in a comment.

Also, `+` is redundant